src/index.html

```diff
@@ -21,7 +21,370 @@
     />
   </head>
 
-  <body>
-    <h1>Catalog</h1>
+  <body class="page_body">
+    <header class="header">
+      <a href="#scr/images">
+        <img
+          src="./images/logo.png"
+          alt="logo"
+          class="header__logo"
+        />
+      </a>
```

Comment:
The href attribute in the `a` tag should lead to the main page of the website or to another relevant page, but not to an image. It's better to use `#` or `/`.